pkg/devfile/adapters/common/command.go

```diff
@@ -279,3 +285,24 @@ func ValidateAndGetDebugDevfileCommands(data data.DevfileData, devfileDebugCmd s
 
 	return debugCommand, nil
 }
+
+// ValidateAndGetTestDevfileCommands validates the test command
+func ValidateAndGetTestDevfileCommands(data data.DevfileData, devfileTestCmd string) (testCommand common.DevfileCommand, err error) {
+	var emptyCommand common.DevfileCommand
+	isTestCommandValid := false
+	testCommand, testCmdErr := GetTestCommand(data, devfileTestCmd)
+	if testCmdErr == nil && !reflect.DeepEqual(emptyCommand, testCommand) {
+		isTestCommandValid = true
+		klog.V(4).Infof("Test command: %v", testCommand.Exec.Id)
+	}
+
+	if !isTestCommandValid {
+		var commandErrors string
+		if testCmdErr != nil {
+			commandErrors = testCmdErr.Error()
```

Comment:
I think this should be `commandErrors += testCmdErr.Error()` like the others above

Reply:
Actually for test command, it only has testCmdErr, so we can use `=`
we use `+=` in the above due to push executes initCmd, runCmd and buildCmd, so need to concatenate the errors.